examples/unixd

```diff
@@ -140,3 +140,14 @@ version = '2'
 
 pam_allowed_login_groups = ["posix_group"]
 
+# Allow extension (mapping) of a local system groups members with members from a
+# kanidm provided group. An example of this is that the local group
+# `libvirt` can have it's membership extended with the members from
+# `virt-admins`. This section can be repeated many times.
```

Comment:
```suggestion
# Kanidm-provided group. An example of this is that the local group
# `libvirt` can have its membership extended with the members from
# `virt-admins`. This section can be repeated many times.
```

Given the "can be repeated many times" can we please include more than one in the example? I know I've tripped over things like this in the past where it's not clear on the exact syntax of *how* to repeat them, or weird edge cases etc. (And ensure it's tested so we can point at something and go "yes this matches what we document")

Reply:
Yep can do